efficientdet/keras/postprocess.py

```diff
@@ -160,17 +160,17 @@ def nms(params, boxes: T, scores: T, classes: T, padded: bool) -> List[T]:
 
   # TF API's sigma is twice as the paper's value, so here we divide it by 2:
   # https://github.com/tensorflow/tensorflow/issues/40253.
-  nms_top_idx, nms_scores, nms_valid_lens = tf.raw_ops.NonMaxSuppressionV5(
+  nms_top_idx, nms_scores, _ = tf.raw_ops.NonMaxSuppressionV5(
       boxes=boxes,
       scores=scores,
       max_output_size=max_output_size,
       iou_threshold=iou_thresh,
-      score_threshold=score_thresh,
+      score_threshold=0,
```

Comment:
We can set score_threshold=0, then filter results when we draw boxes by score.

Reply:
This might slow down the inference latency as NMS needs to process all boxes.
Actually I was thinking the opposite direction: if we filter out boxes based on scores much earlier (in pre_nms), then inference should be faster.
What do you think?

Reply:
I agree.